src/main/java/com/niyati/designpattern/behavioral/chainOfResponsibility/Chain.java

```diff
@@ -0,0 +1,7 @@
+package com.niyati.designpattern.behavioral.chainOfResponsibility;
+
+public interface Chain {
+    public void setNextChain(Chain nextChain);
+
+    public void calculate(Numbers request);
+}
```

Comment:
Add documentation about each of these pattern in the respective package by following https://www.baeldung.com/java-package-info